docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  rabbitmq:
+    container_name: rabbitmq
```

Comment:
It would be convenient to use different container names and ports here to in our docker dev repo so they can run simultaneously without conflicting.

Reply:
done, altered names and ports. Checked that this works with census-rm-docker-dev working alongside